src/libtiled/tile.h

```diff
@@ -252,6 +256,23 @@ inline unsigned Tile::terrain() const
 }
 
 /**
+ * Returns the offset that is applied when drawing the tiles in this
+ * tileset.
```

Comment:
This comment should be updated to "Returns the offset that is applied when drawing the tile."